src/__tests__/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
Why js instead of ts?

Reply:
I added js because I dont think I would be writing typescript in my tests. But anyways, i'll make it a ts file